tests/test_search_open.py

```
@@ -16,3 +23,110 @@ def test_find_anchor_line(tmp_path: Path) -> None:
+
+    monkeypatch.setattr("polylogue.cli.search_cli.open_in_editor", fake_open_in_editor)
+    monkeypatch.setattr("polylogue.cli.search_cli.webbrowser.open", lambda *_a, **_k: pytest.fail("webbrowser.open should not be called"))
+
+    _open_single_search_hit(ui, hit)
+
+    assert called["path"] == target
+    assert called["line"] == 2
+
+
+def test_search_csv_stdout(monkeypatch, capsys) -> None:
+    hit = _hit_for_path(Path("/tmp/conversation.md"))
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/conversation.md"

(S108)